inc/template-tags.php

```diff
@@ -114,7 +114,7 @@ function twentyseventeen_edit_link( $id ) {
 	} elseif ( get_post( $id ) ) {
 		$type = __( 'Post', 'twentyseventeen' );
 	}
-	$link = edit_post_link(
```

Comment:
@codersantosh this pull request looks good now. Some of the code has changed so those links https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L116 & https://github.com/WordPress/twentyseventeen/blob/master/inc/template-tags.php#L58 are no longer showing accurate line numbers.

Fee free to DM me on the Make WordPress slack channel if you need assistance with making another pull request. My slack username is **`@joefusco`**